benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -188,29 +188,33 @@ struct ArborX::AccessTraits<QueriesWithIndex<Queries>, ArborX::PredicatesTag>
   }
 };
 
-template <class TreeType>
+template <typename ExecutionSpace, class TreeType>
```

Comment:
Prefer
```suggestion
template <class ExecutionSpace, class TreeType>
```
for consistency

Reply:
Not changing. If you want to change all things like this, this is for a separate PR, as we already use it a lot in `master`:
```
$ git grep 'template <typename ExecutionSpace' | wc -l
63
```

Reply:
"Local" consistency.  Anyway you may ignore.

Reply:
Ignoring.